pt/scripts/iota_tracker.py

```diff
@@ -34,76 +34,154 @@
     "BEMS9PV9DISPLAY9": PV,
     "BEMS9WT9DISPLAY9": WT,
 }
+# Content of data_field in db
+DB_DATA_TYPE = ("Demand", "ESS", "EV", "PV", "WT")
 
 
-def process_data():
-    # get address from db
+def get_inserted_data(utc_now, db_data_type):
```

Comment:
It's time to use [this plugin](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) to write some docstring.

Reply:
Added in 6c85155.